install.yaml

```diff
@@ -172,4 +176,12 @@ spec:
       - name: bpf-programs
         hostPath:
           path: /sys/fs/bpf
+# TODO(gauravkghildiyal): Try to use embedded PCI DB as an input to https://github.com/jaypipes/ghw
```

Comment:
I see now why I created my own pcidb and didn;t use that library, I don't like this dependency very much

Reply:
You should be able to use embed.FS, yes! should be able to simply set the Chroot setting accordingly. Just keep in mind that you will want to keep that embedded pci.ids.gz file continuously up to date with some sort of automation... it changes often as new product and class codes are added, as you know...